.config/tsaoptions.json

```diff
@@ -0,0 +1,13 @@
+{
+  "areaPath": "DevDiv\\ASP.NET Core",
```

Comment:
Please let me know what area path is correct. @rafikiassumani-msft or @danmoseley may know if @benjaminpetit and @ReubenBond don't.

Reply:
How do we find area paths? It is probably correct

Reply:
This is probably wrong. Should align w/ wherever you get Policheck or CredScan bugs reported. If those aren't turned on, there's a problem in any case. I'm hoping @rafikiassumani-msft and @danmoseley chime in here.